event_generation/locustfile.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+import random
+import json
+
+from locust import User, task, between
+import boto3
+
+import config as cfg
+from tickers import Tickers
+
+
+class StockUser(User):
+    wait_time = between(1, 1)
+
+    def on_start(self):
+        self.kinesis_client = boto3.client("kinesis")
+        self.stream_name = cfg.STREAM_NAME
+
+    @task
+    def send_stock_values(self):
+        for ticker in Tickers:
+            self.kinesis_client.put_record(
+                StreamName=self.stream_name,
+                Data=StockUser.get_data(ticker.value),
+                PartitionKey=str(datetime.now()),
+            )
+
+    @staticmethod
+    def get_data(ticker_value):
+        return json.dumps(
+            {
+                "event_time": datetime.now().isoformat(),
```

Comment:
Hudi table requires a unique identifier. I believe a generator can be used, but for now maybe add an uuid field with `uuid.uuid4()`?